python/lsst/ctrl/platform/slurm/etc/templates/allocation.sh.template

```diff
@@ -9,7 +9,7 @@ if [ $COLLECTOR ]; then
     export RUBIN_COLLECTOR_HOST=$COLLECTOR
 else
     echo "COLLECTOR is empty; using USDF default"
```

Comment:
For this one here we can change to  'using site default' rather than the USDF specific message, 
since its appearance in the logs makes it look like we missed something.

Reply:
Changed to:
```
else
    export RUBIN_COLLECTOR_HOST="tiger3-sumire.princeton.edu"
    echo "COLLECTOR is not set; using default: $RUBIN_COLLECTOR_HOST"
fi
```
How does that read @daues?

Reply:
Oh, I just read your suggestion above - happy to either print the actual value being used, or a more generic message, as you think appropriate.